### ISSUE TITLE
PushPlugin not Maintained

### ISSUE THREAD

**Issue by bisrael** [phonegap-build/PushPlugin#523]
This project has not been maintained in 6+ Months.

Is this plugin dead?

**Comment by vespino** [comment]
Seems to work, but no support is given, at least not to me :(

There also seems to be a lot of forks. What's up with that? I don't know what to do with the problem I'm having.

**Comment by bisrael** [comment]
@vespino - what problem are you having?

**Comment by vespino** [comment]
https://github.com/phonegap-build/PushPlugin/issues/517
https://github.com/phonegap-build/PushPlugin/issues/520


Op 31 mrt. 2015 om 18:43 heeft Blake Israel <notifications@github.com> het volgende geschreven:

@vespino - what problem are you having?

**Comment by clarklight** [comment]
Please help me as well!  I am just really stuck....

I have a problem with onResume(when i click on App icon to resume the App), it clears my notification tray and open the App but no Event call back and the messages just disappears... but other comments says on version 2.4.0 it should work, and when you resume the app there should be a ECB and data pull into the app. Not sure why mine is going weird.

**Comment by justinwp** [comment]
Who has a fork that is more up to date?

**Comment by clarklight** [comment]
Is this the only Push Plugin for Phonegap, i feel that this is the main one and that majority of people uses this one. Hope Phonegap is helping to maintain this one, it doesnt seem this plugin allow Event Call Back for onResume Events.

**Comment by d3orn** [comment]
56 open Pull requests and it stopped working with the latest cordova version on windows phone (only a smaller bug but still). Hope for some activity!

**Comment by TheBosZ** [comment]
For anyone who wants an actually maintained plugin, you can use Telerik's fork: https://github.com/Telerik-Verified-Plugins/PushNotification

**Comment by EddyVerbruggen** [comment]
I can confirm the Telerik one is being actively maintained. It's tailored for their Backend Services solution but it works just as well with any other standard or custom push backend (as you were used to with this one).

**Comment by jlsync** [comment]
The Telerik fork has removed all the useful documentation from the README. 

Perhaps @wildabeast or @EddyVerbruggen @goya @aogilvie  could update this issue, or update this repository's description or readme.

@macdonst has a fork here  https://github.com/phonegap/phonegap-plugin-push  (e.g.  phonegap/phonegap-plugin-push#2 ) - is this Adobe's/Phonegap's supported version going forward?

**Comment by mcm-ham** [comment]
Thanks @EddyVerbruggen I've been struggling getting receiving of notifications to work on Android thinking I was doing something wrong but as soon as I switched to the Telerik fork it worked straight away.

**Comment by EddyVerbruggen** [comment]
@mcm-ham that's awesome, thanks for sharing that!

**Comment by macdonst** [comment]
@jlsync Yes, the intention is the phonegap-push-plugin repo will be maintained going forward with a slightly easier to use API.
